File: 프로그래머스/lv0/120825. 문자 반복 출력하기/문자 반복 출력하기.java
class Solution {
    public String solution(String my_string, int n) {
        
        String answer = "";
        String[] answerArr = my_string.split("");
        
        for (int i=0 ; i< my_string.length(); i++){
            answer += answerArr[i].repeat(n);
        }
        return answer;
    }
}

File: 프로그래머스/lv0/120825. 문자 반복 출력하기/문자 반복 출력하기.cs
using System;

public class Solution {
    public string solution(string my_string, int n) {
        string answer = "";

        Char[] answerArr = my_string.ToCharArray();
        
        for (int i=0 ; i< answerArr.Length; i++){
            
            for(int j=0; j<n; j++){
              answer += answerArr[i]; 
            }

        }
        return answer;
    }
}
